Twilio-org/phonebank
# Questions id view

# Description
- Created view for viewing a question `admin/questions/:id`
- Linked the view button in the admin list views
- Added material icons to the index for now in order to use in this particular view
- Created tests

### Notes
- Noticed some differences in reducer data set up so I uncommented the handleClick line for now
    - my **fetchQuestion(id)** action is sending the question to the current_question reducer with the responses in an array e.g. ['yes','no', 'maybe']
    - if I leave the handleClick method on, the current_question set up on the admin list views has the question data stored differently (string).

Should I make a separate ticket to consolidate this? I think having the responses already structured as an array in the store is better than having to parse the string every time we want to render.

## Test plan
- go to /admin/questions
- click on 'view' for any of the questions (also is working if you view a script)
- run `npm run test:client` to see tests run
